packages/pds/src/api/app/bsky/feed/setVote.ts

```diff
@@ -12,13 +12,14 @@ export default function (server: Server) {
     auth: ServerAuth.verifier,
     handler: async ({ auth, input, res }) => {
       const { subject, direction } = input.body
-      const { db } = locals.get(res)
+      const { db, services } = locals.get(res)
 
       const requester = auth.credentials.did
       const authStore = await locals.getAuthstore(res, requester)
       const now = new Date().toISOString()
 
       const voteUri = await db.transaction(async (dbTxn) => {
+        const repoTxn = services.repo.using(dbTxn)
```

Comment:
I don't think this is bad, but just thinking through possible footguns. I feel like it's very easy to forget to do `.using(tx)` & accidentally fire off non-tx queries from within a tx.

Almost wonder if we should not pass around services that are hooked up to the DB & always make the route instantiate them (either with a non-tx db or with a dbTxn).

Nearly all of our logic is held in txns anyway so i don't think this would increase boilerplate code much